changelog/1.19.0.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "1.19.0"
+description: "Released on 05/19/2021"
+---
+
+### Breaking changes ❗
+
+- infra: Workspace assets created by Coder previously located at `/opt/coder`,
+  such as code-server and coder-cli, have been moved to `/var/tmp/coder`
+
+### Features ✨
+
+- web: Introduced resource quotas to organizations. Resource quotas define the
+  maximum resource utilization allowable for each member within an organization
+  for which such quotas are enabled. Modify them at **Manage** >
+  **Organizations** > **Edit Organization**.
+- web: Added site and organization workspace as code templates.
+- web: Added administrative control over dev URL access levels.
+- web: The CVM option when creating a custom environment is now selected by default.
+- web: Improved display and support for errors returned by an IdP during sign-in
+- infra: Changed permissions of `/home/coder/` from _755_ to _744_
+- infra: Workspace builds are now validated when setting policy templates
+- infra: Added client certificate authentication support
+
+### Bug fixes 🐛
+
+- web: Fixed issue where the **Edit Workspace** dialog showed the resource
+  selectors as disabled while still allowing modifications
+- web: Fixed issue where the **Edit Workspace** dialog displayed resource
+  allocation incorrectly if the workspace was initially built using a template
+- web: Updated validation logic so that workspaces with invalid names can no
+  longer be created
+- infra: Fixed issue causing workspace autostart to stop or fail intermittently
+- web: Fixed issue in the **Audit Logs** preventing users from clearing the
+  filters
+
+### Security updates 🔐
```

Comment:
I guess we should add a section here about updates to certificate handling support, particularly that (a) we now have debug logs for certificates that we load, and (b) we can reload certificates with a SIGHUP is received. I'm not sure whether these belong under security or features, maybe features.

Maybe: "infra: Add debug logs showing TLS certificates, and reload certificates when a SIGHUP signal is received" ?